allensdk/brain_observatory/multi_stimulus_running_speed/multi_stimulus_running_speed.py

```diff
@@ -1,19 +1,20 @@
-import numpy as np
 import pandas as pd
```

Comment:
Could you please add a note at the top of the module that this module is specifically for writing the multi stimulus running speed?